src/app/pages/private/collection/collection.page.html

```diff
@@ -36,5 +40,6 @@
   <app-rarity-slider
     [rarities]="rarities()"
     [selectedRarity]="selectedRarity()"
+    [isResetSelectedValue]="isSearchActive()"
```

Comment:
slider should not handle which value to show. Pass undefined in case if search is avtice instead of last selected rarity